modules/base/rendering/renderabletimevaryingsphere.cpp

```diff
@@ -67,157 +67,157 @@ namespace {
 
 namespace openspace {
 
-documentation::Documentation RenderableTimeVaryingSphere::Documentation() {
-    return codegen::doc<Parameters>("base_renderable_time_varying_sphere");
-}
-
-RenderableTimeVaryingSphere::RenderableTimeVaryingSphere(
-                                                      const ghoul::Dictionary& dictionary)
-    : RenderableSphere(dictionary)
-    , _textureSourcePath(TextureSourceInfo)
-{
-    const Parameters p = codegen::bake<Parameters>(dictionary);
-
-    _textureSourcePath = p.textureSource.string();
-}
-
-bool RenderableTimeVaryingSphere::isReady() const {
-    return RenderableSphere::isReady() && _texture;
-}
-
-void RenderableTimeVaryingSphere::initializeGL() {
-    RenderableSphere::initializeGL();
-
-    extractMandatoryInfoFromSourceFolder();
-    computeSequenceEndTime();
-    loadTexture();
-}
-
-void RenderableTimeVaryingSphere::deinitializeGL() {
-    _texture = nullptr;
-    _files.clear();
-
-    RenderableSphere::deinitializeGL();
-}
-
-void RenderableTimeVaryingSphere::extractMandatoryInfoFromSourceFolder() {
-    // Ensure that the source folder exists and then extract
-    // the files with the same extension as <inputFileTypeString>
-    namespace fs = std::filesystem;
-    const fs::path sourceFolder = absPath(_textureSourcePath);
-    if (!std::filesystem::is_directory(sourceFolder)) {
-        throw ghoul::RuntimeError(
-            "Source folder for RenderableTimeVaryingSphere is not a valid directory"
-        );
+    documentation::Documentation RenderableTimeVaryingSphere::Documentation() {
```

Comment:
The indentation of all the functions is this file need to be fixed. There should be no indentation before these functions